### DISCUSSION TITLE
Can I store GitHub Action YAML Files to non-default branches?

### DISCUSSION THREAD

**Discussion by ymyn01** [community/community#120931] [New to GitHub]
Will these Actions be recognized and run automatically, like those stored in the default branch?

**Comment by Arifulhaque313** [comment]
reqest 1

**Comment by Arifulhaque313** [comment]
requst 2

**Comment by gurjeetsinghvirdee** [comment]
Hey @ymyn01 

You can store GitHub Action YAML files in non-default branches. While GitHub automatically looks for workflow files in the `.github/workflows` directory of your default branch (usually main or master), you can also place them in other branches or directories within your repository. Just create the `.github/workflows` directory in the desired branch and put your YAML files there. GitHub will recognize workflows in any branch as long as they are located within this directory.
